plugins/source/azure/docs/tables/README.md

```diff
@@ -224,13 +223,27 @@
 - [azure_mysql_servers](azure_mysql_servers.md)
   - [azure_mysql_server_configurations](azure_mysql_server_configurations.md)
 - [azure_network_express_route_gateways](azure_network_express_route_gateways.md)
+- [azure_security_assessments](azure_security_assessments.md)
 - [azure_security_pricings](azure_security_pricings.md)
+- [azure_security_settings](azure_security_settings.md)
```

Comment:
Also one for `azure_security_contacts`?

Reply:
Thank you for noticing it. I have removed contacts  from plugins/source/azure/resources/plugin/plugin.go since they have broken api for now
link to the issue in sdk https://github.com/Azure/azure-sdk-for-go/issues/19740